appveyor.yml

```diff
@@ -4,21 +4,21 @@ environment:
   - MSYS_BITS: 64
     TARGET: x86_64-pc-windows-msvc
     CHECK: check
-    CONFIGURE_ARGS: --enable-llvm-assertions --enable-debug-assertions
+    CONFIGURE_ARGS: --enable-vendor
```

Comment:
Why are all these configurations remove assertions? And why isn't this patch adding `--enable-vendor` to the travis configs?